designs/virtualization/4.x/create-vm-dropdowns/create-vm-dropdowns.md

```diff
@@ -0,0 +1,137 @@
+---
+parent: Virtualization
+version: 4.x
+---
+
+# create VM dropdowns
+
+This item will cover various updates to the VM creation wizard
+- Adding descriptions to the dropdown fields (using this pattern https://github.com/patternfly/patternfly-design-kit/issues/218)
+- Renaming fields
+- Reordering fields
+- Adding 'Operating System source' step
+
+
+![Create VM - General-default](img/1-0.jpg)
+Landing on the General step, Default mode.
+Section titles added, and fields reorgenized and renamed.
+Any OS image source related fields are mofed to the next step.
+
+'General' section:
+- 'Name'
+- 'Description'
+'Resources & configuration' section'
+- 'Operating system'
+- 'Workload type' (previously Workload profile)
+- 'Flavor (CPU and Memory)' (previously only 'flavor')
+
+
+## General step
+
+![Create VM - General](img/1-1.jpg)
+Upon selecting an operating system, the Workload type and Flavor (CPU and Memory) fields will be enabled and their defaults will be selected.
+
+![Create VM - General - Workload type](img/1-1-1.jpg)
+Workload type options
+Server is the default
+
+![Create VM - General - flavor](img/1-1-2.jpg)
+Flavor options.
+Medium is the default
+
+## Operating system source step (new)
+
+![Create VM - General](img/1-2.jpg)
+Operating system source step,landing mode.
+
+![Create VM - General](img/1-2-1.jpg)
+'Boot operating system via' field options
+
+![Create VM - General](img/1-2-2.jpg)
+Selecting an option from the 'Boot operating system via' field will expose below all the fields required to upload the image and create the correlating disk/NIC.
+For example: on selecting the 'URL' option, additional fields will appear below.
+These fieldss are required in order to point to the desired URL asl well as creating a disk via PVC that will containt the image on that URL.
+The fields are basically mapped 1:1 to the Disk modal, with the URL option selected for 'Disk content' field.
+
+An Advanced (settings) drawer is available here as well, containing the same options as the Advanced options on the disk modal.
+
+![Create VM - OS disk on list](img/1-3.jpg)
```

Comment:
two little nits on the alert
Could we just say: "Rootdisk" contains the operating system image set on the operating system source"
These alerts should be info alerts right? I think the bell alert is reserved for very generic content, this feels more like info to me.